src/fetchIdentifiers.ts

```diff
@@ -0,0 +1,113 @@
+import { GraphQLClient } from 'graphql-request';
+import { CREATE_IDENTIFIER, IDENTIFIERS, NEW_IDENTIFIER_TYPES } from './gqls';
+import keyBy from 'lodash/keyBy';
+import uniq from 'lodash/uniq';
+import flatten from 'lodash/flatten';
+import { TranscendInput } from './codecs';
+import difference from 'lodash/difference';
+import { logger } from './logger';
+import colors from 'colors';
+import { mapSeries } from 'bluebird';
+
+export interface Identifier {
+  /** ID of identifier */
+  id: string;
+  /** Name of identifier */
+  name: string;
+}
+
+const PAGE_SIZE = 20;
+
+/**
+ * Fetch all identifiers in the organization
+ *
+ * @param client - GraphQL client
+ * @returns The contents of the yaml file, type-checked
+ */
+export async function fetchAllIdentifiers(
+  client: GraphQLClient,
+): Promise<Identifier[]> {
+  const identifiers: Identifier[] = [];
+  let offset = 0;
+
+  // Try to fetch an enricher with the same title
+  let shouldContinue = false;
+  do {
+    const {
+      identifiers: { nodes },
+      // eslint-disable-next-line no-await-in-loop
+    } = await client.request(IDENTIFIERS, {
+      first: PAGE_SIZE,
+      offset,
+    });
+    identifiers.push(...nodes);
+    offset += PAGE_SIZE;
+    shouldContinue = nodes.length === PAGE_SIZE;
```

Comment:
Could this fetch the `totalCount` and loop until `identifiers.length === totalCount`?

Reply:
it probably could, but i think doing it this way may be a bit sturdier in the corner case that someone create a new identifier mid-pagination.